mix.exs

```diff
@@ -276,7 +276,7 @@ defmodule Ash.MixProject do
       {:ecto, "~> 3.7"},
       {:ets, "~> 0.8.0"},
       {:decimal, "~> 2.0"},
-      {:picosat_elixir, "~> 0.2"},
+      {:picosat_elixir, "~> 0.2.3"},
```

Comment:
This probably isn't necessary for this PR, right?

Reply:
I think we should stay without the patch dependency. Users will get the latest fixes by updating the dep, individual patches don't need to update their dependency on other packages when they add fixes.

Reply:
huh. That is pretty interesting...I wonder why that is?